tests/RobotsTxtTest.php

```diff
@@ -128,4 +128,60 @@ public function the_disallows_user_agent_check_is_case_insensitive()
         $this->assertFalse($robots->allows('/no-agents', 'UserAgent007'));
         $this->assertFalse($robots->allows('/no-agents', strtolower('UserAgent007')));
     }
+
+    /** @test */
+    public function it_can_handle_multiple_user_agent()
```

Comment:
Can this test be made smaller? Due to the sheer length, it's hard to see what is being tested. Can you test the same behaviour with less user agents?

Reply:
I split tests and remove some user-agents